Makefile

```diff
@@ -54,6 +54,10 @@ build:
 	$(GOBUILD) -o ./bin/$(BUILD_TARGET_IOCTL) -v ./cli/ioctl
 	$(GOBUILD) -o ./bin/$(BUILD_TARGET_MINICLUSTER) -v ./tools/minicluster
 
+.PHONY: buildcli
+buildcli:
```

Comment:
let's name it ioctl, "make ioctl" should do it.

Reply:
done